Blammo/README.lhs

```diff
@@ -44,6 +44,8 @@ import Control.Monad.Reader (MonadReader, ReaderT (runReaderT))
 
 ```haskell
 import Blammo.Logging.Simple
+import Blammo.Logging.LogSettings
+import Blammo.Logging.ThreadContext
```

Comment:
I dunno, I might consider moving some of the re-exporting to `.Simple`, so this doesn't change. I thought of the name "simple" here as "all you need" or "batteries included", not as in "minimal". Seeing additional imports needed in the documentation tells me it's no longer completing the brief, so to speak.